webapp/api/advantage.py

```diff
@@ -11,7 +11,12 @@
 
 
 class UnauthorizedRequest(Exception):
-    pass
+    def __init__(self, url=None, payload=None):
+        self.url = url
+        self.payload = payload
+
+    def as_dict(self):
+        return {"url": self.url, "payload": self.payload}
```

Comment:
Another option would be to simply do `response.raise_for_status` lower down and then catch the resulting `HTTPError`, which would presumably contain all the information you need when it's passed through to sentry?

Reply:
Well it's available at `HTTPError.response.json` or something, so you could simply update your code at the point where you `captureException` to extract this data.

Reply:
Ah, sorry I though you were suggesting something different. I'll update